### ISSUE TITLE
Ordering Problem

### ISSUE THREAD

**Issue by thomaskkrachten** [joschahenningsen/TUM-Live#1149]
Wenn oldest first ausgewählt wird, dann ist die Auflistung July, Juni, Mai, April und wenn newest first dann ist sie umgekehrt. Die Auflistung der Videos passt zu der gewählten Ansicht.

**Comment by MariusAlbrecht** [comment]
can confirm. Ordering is the wrong way around.
The html element in question has the css path `html.dark.juheove.ykhplhs body.h-screen.flex.flex-col.items-stretch.tum-live-bg main#content.flex.grow.h-full.overflow-y-scroll article.text-3.p-4.grow article.tum-live-course-view div.grid.gap-x-8.xl:grid-cols-3.grid-cols-1 section.tum-live-course-view-item.col-span-full section article.flex.flex-col`
and html code `<article class="flex flex-col" : class="{'flex-col-reverse' : isOldestFirst()>`

The problem _seems_ to be fixed by replacing `isOldestFirst()` with `!isOldestFirst()` as is already done [in the code](https://github.com/TUM-Dev/gocast/blob/6eb96ea6b6ce33a7a3ccc92f3e3219e93eb256b1/web/template/home.gohtml#L434)

that bit of code just doesnt seem to be online (yet?)

**Comment by MatthiasReumann** [comment]
> can confirm. Ordering is the wrong way around. The html element in question has the css path `html.dark.juheove.ykhplhs body.h-screen.flex.flex-col.items-stretch.tum-live-bg main#content.flex.grow.h-full.overflow-y-scroll article.text-3.p-4.grow article.tum-live-course-view div.grid.gap-x-8.xl:grid-cols-3.grid-cols-1 section.tum-live-course-view-item.col-span-full section article.flex.flex-col` and html code `<article class="flex flex-col" : class="{'flex-col-reverse' : isOldestFirst()>`
> 
> The problem _seems_ to be fixed by replacing `isOldestFirst()` with `!isOldestFirst()` as is already done [in the code](https://github.com/TUM-Dev/gocast/blob/6eb96ea6b6ce33a7a3ccc92f3e3219e93eb256b1/web/template/home.gohtml#L434)
> 
> that bit of code just doesnt seem to be online (yet?)

Yes exactly. The problem should be fixed after #1145 is deployed. Thanks for the issue anyways!

**Comment by joschahenningsen** [comment]
Just deployed the newest version, should be all good now.

**Comment by alexanderstephan** [comment]
Ah, I think it is still broken. It is now only sorted by month, not respecting the year, i.e., January 2023 is before October 2022. E.g, see https://live.rbg.tum.de/?year=2022&term=W&slug=jms&view=3.

**Comment by MatthiasReumann** [comment]
> https://live.rbg.tum.de/?year=2022&term=W&slug=jms&view=3

Seems to be right. I will try to fix this.
